src/client/datascience/notebook/intellisense/completionProvider.ts

```diff
@@ -76,7 +77,10 @@ export class NotebookCompletionProvider implements CompletionItemProvider {
             })
         ]);
         if (!result) {
+            traceInfoIf(isCI, `Notebook completions not found.`);
             return [];
+        } else {
+            traceInfoIf(isCI, `Completions found, filtering the list: ${JSON.stringify(result)}.`);
```

Comment:
I think we should just turn this into `traceIfCI`, lol, feels kinda redundant to always pass in `isCI`. What do you think.
We might have a few places where we'd have some other conditions being passed in, but i think traceIfCI` or `traceOnCI` is simpler.
Absolutely no need to make that change, just a thought

Reply:
Yeah sounds like a good idea. I'll add a debt issue